deployment/ansible/group_vars/production.example

```diff
@@ -22,6 +22,7 @@ app_version: "0.0.0"
 app_domain_name: "your.driver-domain.here"
 
 ## DOCKER SETTINGS
+docker_image_tag: "{{app_version}}"
```

Comment:
This seems like something that might fit in the `ADVANCED` section at the bottom, since it's just a reassignment of an already provided value that I suspect the user won't need to set themselves, like what we do with `allowed_host`:
https://github.com/WorldBank-Transport/DRIVER/blob/3ae5b47e182e4f2074beef0a0a0d2f31d80084a4/deployment/ansible/group_vars/production.example#L260-L263